index.html

```diff
@@ -10,18 +10,13 @@
   </head>
 
   <body>
-    <header>
-      <nav class="header">
-        <ul class="nav-items">
-          <li class="logo"><a>Megha</a></li>
-          <li class="menu"><i class="fa-solid fa-bars"></i></li>
-        </ul>
-      </nav>
+    <header class="headercontainer">
+      <div class="logo">Megha</div>
+          <div><i class="fa-solid fa-bars hambrgricon"></i></div>
     </header>
 
     <section class="main-section">
       <div class="FirstShape">
-        <img src="/images/Ellipse8.png" alt="shapes"/>
       </div>
       <div class="first">
         <img class="half-ellipse" src="/images/Ellipse11.png" alt="shapes"/>
```

Comment:
- [ ] Same comment for this shape, please remove all shapes from your HTML and use a single background that has all the shapes as I showed you above! 😌